src/Lane.cpp

```diff
@@ -6,3 +6,10 @@ int Lane::getNumberOfGames()
 {
     return 0;
 }
+
+
+std::ifstream Lane::open(std::string pathToFile)
+{
+    std::ifstream laneFile;
```

Comment:
Brakuje jeszcze przekazania ścieżki do pliku to ifstream:
`std::ifstream laneFile(pathToFile);`